litmus/libdir/_aarch64/kvm_auth.c

```diff
@@ -0,0 +1,104 @@
+// read the SCTLR_EL1 status register
```

Comment:
Hi @RemyCiterin. Having non-static functions in a file that is included in the C source of tests is not possible. Namely, with option `-driver C` all tests are bundled together in a unique C executable and the various instances of the functions below will clash.

I guess thet the best solution is to link the test executable with `kvm_auth.o`. I will suggest something soon.

Reply:
I had a different solution in mind : do not insert the `kvm_auth.c` code in test code, but link with it instead. Doing so I discovered that kvm-unit-tests is not strictly necessary for the simplest authentification modes. And indeed one can run simple PAC tests with user qemu. I have implemented both enhancements here: 1a8bc7ba1913c70b1acd551443db46ee7f1e4491. Would you consider cherry-picking this commit here?